### PR TITLE
Lock index - overall edits - (edits PR 2020)

### PR DESCRIPTION
**Pull request by Catografix**
Reduced image size and moved to repo.
Fixed links to Browserify and webpack.
Created an include for the Version disclaimer - should be linked to from all the new v10 docs instead of editing each.
Moved disclaimer below image.
Changed first subhead to reflect page contents since it becomes the page title.

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by Catografix** [comment]
@jeffreylees Please review. If you like the version include, I can add it to the other docs, replacing the hard-coded in each.

File: articles/libraries/lock/index.md [hunk 1]
```diff
@@ -143,37 +141,37 @@ var client = new Auth0({
 });
 ```
 
-## Start using Lock!
+## Start using Lock
 
 ${lockSDK}
```
Comment (Catografix) [inline articles/libraries/lock/index.md:146]: Where is the source for this ${lockSDK}? It needs editing!!

**Comment by Catografix** [comment]
Preview OK: https://auth0-docs-content-pr-2066.herokuapp.com/docs/libraries/lock

**Comment by jeffreylees** [comment]
Disclaimer include is good because at some point we'll change or remove it, might as well do that now. Could use the include method from here on in on version changes, and the same when v9 is EOL, on the v9 docs.

What needs edited in `${lockSDK}` ?

**Comment by Catografix** [comment]
@jeffreylees 
There are many ways of integrating Auth0 into your site. 
Choose from the methods below to generate a preview and code snippets.
This is how it will look in a browser:
Copy this HTML/JS code into your website:

**Pull request merged**